dashboard/src/components/AppView/DeploymentsTable/DeploymentItem.test.tsx

```diff
@@ -24,13 +35,18 @@ context("when fetching deployments", () => {
       component: DeploymentItem,
       props: {
         deployment,
-        getDeployment: jest.fn(),
+        watchDeployment: jest.fn(),
       },
     });
 
     it("displays the name of the deployment", () => {
       const wrapper = shallow(
-        <DeploymentItem deployment={deployment} name="foo" getDeployment={jest.fn()} />,
+        <DeploymentItem
```

Comment:
we can save a few lines of code if we store this props as `defaultProps` and we use that around.

Reply:
good shout :+1: